<?xml version="1.0"?>

<launch>
  <master auto="start"/>

  <!--  -->
  <node pkg="eurobot2015_petit_ai" type="petit_objective_manager" respawn="false" name="petit_objective_manager" output="screen">
	<param name="map_name" type="str" value="/petit_map" />

        <remap from="/ROBOT/best_objective" to="/DEBUG/best_objective" />
        <remap from="/ROBOT/debug_path" to="/DEBUG/debug_path" />
        <remap from="/ROBOT/get_robot_pose" to="/PETIT/get_robot_pose" />
        <remap from="/ROBOT/get_distance" to="/PETIT/get_distance" />
        <remap from="/ROBOT/get_objective" to="/PETIT/get_objective" />
        <remap from="/ROBOT/delet_objective" to="/PETIT/delet_objective" />
        <remap from="/ROBOT/update_priority" to="/PETIT/update_priority" />
        <remap from="/GENERAL/color" to="/GENERAL/color" />
  </node>

  <!--  -->
  <node pkg="common_smart_ai" type="eurobot_timer_manager" respawn="false" name="petit_timer_manager" output="screen">
        <remap from="/remain" to="/GENERAL/remain" />
        <remap from="/start_match" to="/GENERAL/start" />
        <remap from="/stop_match" to="/GENERAL/stop_match" />
        <remap from="/pause_nav" to="/PETIT/pause_nav" />
  </node>
  
</launch>
